dev/merge_spark_pr.py

```diff
@@ -331,6 +331,9 @@ def choose_jira_assignee(issue, asf_jira):
                     assignee = asf_jira.user(raw_assignee)
                 asf_jira.assign_issue(issue.key, assignee.key)
                 return assignee
+        except KeyboardInterrupt:
+            traceback.print_exc()
+            sys.exit(-1)
```

Comment:
any reason to do this, rather than just

```
except KeyboardInterrupt:
  raise
except:
  ...
```
?  That seems to do the right thing in a small test I tried, letting you break out of the loop:

```
while True:
  try:
    x = raw_input("enter a number: \n")
    print int(x)
  except KeyboardInterrupt:
    raise
  except:
    print "oops, entered " + x
```

`sys.exit` makes it impossible for an wrapping function to do its own handling.

Reply:
nope. I don't have a special reason for it - just found that's used in this script and thought it's better to make sure exiting. The root exception handling already catches all the exceptions and looks the exception is being handled for now.
But yup let me change and test it in other PRs or this PR :-). not a big deal.